nx-X11/programs/Xserver/mi/miexpose.c

```diff
@@ -610,8 +610,8 @@ miPaintWindow(WindowPtr pWin, RegionPtr prgn, int what)
 	draw_x_off = drawable->x;
 	draw_y_off = drawable->y;
 
-	tile_x_off = 0;
-	tile_y_off = 0;
+	tile_x_off = pWin->drawable.x - draw_x_off;
```

Comment:
This backport might be meant well, but won't actually fix anything. With the current code, the behavior will at best be unchanged (since the new expressions should evaluate to zero anyway) and in the worst case break completely due to underflows.

For this to actually work correctly, we'll need a partial or full backport of [e4d11e58ce349dfe6af2f73ff341317f9b39684c](https://cgit.freedesktop.org/xorg/xserver/commit/?id=e4d11e58ce) first - but that one is HUGE!